src/modules/members/ui/components/columns.tsx

```
@@ -8,7 +8,15 @@ export const columns: ColumnDef<MemberGetOne>[] = [
     accessorKey: "status",
     header: "Status",
     cell: ({ row }) => {
-      return <p className="text-gray-400 italic">N/A</p>;
+      return (
+        <span
+          className={
+            true ? "font-medium text-green-600" : "font-medium text-red-600"
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)